lib/dashboard-template.hbs

```diff
@@ -485,6 +485,7 @@ body, html {
                     </div>
                 </div>
                 {{#with request}}
+                {{#unless @root.skipSensitiveData}}
```

Comment:
Can you also update the other templates, please?

Reply:
Forgot! Brb :)